activity/index.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html>
+
+<head>
+    <title>Community Activity</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.bundle.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="/static/timeago.js"></script>
+    <script src="/static/charts.js"></script>
+    <style>
+    canvas {
+        -moz-user-select: none;
+        -webkit-user-select: none;
+        -ms-user-select: none;
+    }
+    </style>
+</head>
+
+<body>
+    <h1>Community Activity</h1>
+    <script>loadTimeElements()</script>
```

Comment:
Why not put it inside the same script tag below?

Reply:
Actually, this is a remain of when I was serving the page using django. It's no longer possible to display the time stamp(since it's not generated via django)